### DISCUSSION TITLE
第一页页码显示是第二页请问怎么改呢

### DISCUSSION THREAD

**Discussion by HaoKang-Timmy** [TheNetAdmin/zjuthesis#328] [General]
您好，我用了最新的9.1.0版本。我在overleaf上编译发现我的绪论部分（也就是第一部分的第一页）已经是第三页的页码了。
请问我有没有什么办法改成第一页。

**Comment by qtxyq** [comment]
请问lz解决这个问题了吗，我也遇到了这个问题
